packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6079,14 +6102,14 @@ export default class Meeting extends StatelessWebexPlugin {
         name: 'client.mediaquality.event',
         options: {
           meetingId: this.id,
-          networkType: options.networkType,
```

Comment:
Not sure if I correctly understand our intentions here. We can keep it in `options.networkType = options.data.networkType` or have it only in `payload` to avoid duplication. With that logic, I have been leaning towards the second option.

Reply:
@brycetham Yeah, I thought the same, so I will keep it for now. I think I'll add a comment later for the SDK team to discuss this. Thanks

Reply:
@brycetham changed this part with networkType.